python/pylibcugraph/pylibcugraph/experimental/__init__.py

```diff
@@ -71,3 +71,6 @@
 
 from pylibcugraph.katz_centrality import EXPERIMENTAL__katz_centrality
 katz_centrality = experimental_warning_wrapper(EXPERIMENTAL__katz_centrality)
+
+from pylibcugraph.triangle_count import triangle_count
```

Comment:
Triangle count is a new pylibcugraph binding, so it wouldn't have been in experimental for a release. Is the promoted from experimental warning necessary in that case / having it also in experimental?